### PR TITLE
Add polls to web page

### PR DESCRIPTION
**Pull request by eash130**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/PollServlet.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles the poll on the web page. */
+@WebServlet("/poll")
+public class PollServlet extends HttpServlet {
+
+  private static final String POLL_NAME = "pollName";
+  private static final String POLL_RESPONSE = "pollResponse";
+  private Map<String, Map<String, Integer>> polls = new HashMap<>();
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/PollServlet.java]: Is your plan to store this data in a database?

File: portfolio/src/main/java/com/google/sps/servlets/PollServlet.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, Integer> poll = polls.get(request.getParameter(POLL_NAME));
+    if (poll == null) {
+      poll = new HashMap<>();
+      polls.put(request.getParameter(POLL_NAME), poll);
+    }
+    String pollResponse = request.getParameter(POLL_RESPONSE);
+    if (!pollResponse.isEmpty()) {
+      int currentVotes = poll.containsKey(pollResponse) ? poll.get(pollResponse) : 0;
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/PollServlet.java]: You might want to standardize this data for capitalization (equalsIgnoreCase) as a simple optimization.

**Review by prabenzom — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/PollServlet.java (hunk 1 above)
Comment (eash130) [reply portfolio/src/main/java/com/google/sps/servlets/PollServlet.java]: Fixed. (finally)

File: portfolio/src/main/java/com/google/sps/servlets/PollServlet.java (hunk 2 above)
Comment (eash130) [reply portfolio/src/main/java/com/google/sps/servlets/PollServlet.java]: Done.

**Review by alygerharter — APPROVED** [review]

**Review by prabenzom — APPROVED** [review]

**Pull request merged**